### PR TITLE
cis changes

### PR DESCRIPTION
**Pull request by leodotcloud**

### CODE REVIEW

**Comment by maggieliu** [comment]
Issue:
Profiles: rancher/rancher#25325

File: charts/rancher-cis-benchmark/v0.0.1/values.yaml [hunk 1]
```diff
@@ -6,9 +6,12 @@ replicaCount: 1
 
 # if owner is specified, it's used for the name of the configmap for results
 owner: ""
-# skipConfigMapName is used to specify the name of cm where the skip info is stored
-# skip has higher precedence than what's specified in the configmap
-skipConfigMapName: ""
+# userSkipConfigMapName is used to specify the name of cm where user skip info is stored
+userSkipConfigMapName: ""
+# defaultSkipConfigMapName is used to specify the name of cm where default skip info is stored
+defaultSkipConfigMapName: ""
```
Comment (prachidamle) [inline charts/rancher-cis-benchmark/v0.0.1/values.yaml:12]: Where is the info for the tests that Rancher skips by default stored? Does Rancher skip any tests by default?

Comment (leodotcloud) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:12]: This info is stored in k-d-m

Comment (leodotcloud) [reply charts/rancher-cis-benchmark/v0.0.1/values.yaml:12]: https://github.com/rancher/kontainer-driver-metadata/pull/133/commits/9716f296470035b6558861d479e3bab570340672#diff-47fd5e1cb40a28c9140668e63dd3127dR26

**Review by deniseschannon — APPROVED** [review]

**Pull request merged**